v2/fleet-local/control/zk-exhibitor@.service

```diff
@@ -25,6 +25,7 @@ ExecStart=/usr/bin/sh -c "sudo /usr/bin/docker run \
   -e S3_BUCKET=`etcdctl get /zookeeper/s3_exhibitor_bucket` \
   -e S3_PREFIX=`etcdctl get /zookeeper/s3_exhibitor_prefix` \
   -e ZK_ENSEMBLE_SIZE=`etcdctl get /zookeeper/ensemble_size` \
+  -e ZK_PASSWORD=`etcdctl get /zookeeper/ZK_PASSWORD` \
```

Comment:
I tried setting it to blank via jenkins:

`etcdctl get zookeeper/ZK_PASSWORD == zkpassword$
etcdctl set zookeeper/ZK_PASSWORD ''
Error:  value required`